store/evm_aux/store.go

```diff
@@ -49,51 +72,47 @@ type ChildTxRef struct {
 }
 
 type EvmAuxStore struct {
-	db *leveldb.DB
-}
-
-func NewEvmAuxStore(db *leveldb.DB) *EvmAuxStore {
-	return &EvmAuxStore{db: db}
+	db          dbm.DB
+	batch       dbm.Batch
```

Comment:
Why is this batch here? Just create one in `CommitReceipts`.

Reply:
Done